src/UnityExtension/Assets/Editor/GitHub.Unity/UI/HistoryView.cs

```diff
@@ -321,8 +323,14 @@ public void OnEmbeddedGUI()
 
                 GUILayout.FlexibleSpace();
 
+                GUI.enabled = currentRemote != null;
```

Comment:
Should get into the habit of using [EditorGUI.Begin](https://docs.unity3d.com/ScriptReference/EditorGUI.BeginDisabledGroup.html)/[EndDisabledGroup](https://docs.unity3d.com/ScriptReference/EditorGUI.EndDisabledGroup.html) rather than setting GUI.enabled directly. Those utility methods handle nesting properly. (There's also [EditorGUI.DisabledScope](https://docs.unity3d.com/ScriptReference/EditorGUI.DisabledScope.html), for an IDisposable way of doing things.)